support/build/extensions/no-debug-non-zts-20121212/sundown

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+dep_name=$(basename $BASH_SOURCE)
+
+source $(dirname $BASH_SOURCE)/../pecl
```

Comment:
I'm not sure what you mean. Do you mean a relative path instead of an absolute one?

``source ../../pecl``